Tests/Shopware/Components/Plugin.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace Shopware\Components;
```

Comment:
i think we dont need this because we have shopware as dev dependency right?

Reply:
The constants in ShopwarePaymentMethod do refer to MollieShopware. This class does extend from Plugin. The kernel in the unit tests is not loaded correctly. 

I haven't checked the pipeline, but when using the shopware release instead of the repository the '\Tests\bootstrap.php' is missing. For just having this one class the whole autoloading in the unit tests has to be changed.

Reply:
but it works for me?

<img width="1109" alt="shopware-5_–_ShopwarePaymentMethodTest_php" src="https://user-images.githubusercontent.com/5579326/112655467-b3ccbf80-8e50-11eb-93ef-2be4f54152b8.png">

Reply:
I've rebased after test implementation. Before it was not working. Yes, wo don't need it anymore.

Reply:
perfect, thx, the dev dependency has been added a while ago